Arion/include/Arion/ExpandingPolytopeAlgorithm.hpp

```diff
@@ -34,40 +34,37 @@ void BlowUpPolytope(arion::intersection::gjk::Simplex& simplex, ShapeA const& aS
 
     if (1 == simplex.size)
     {
-        glm::dvec3 const A0 = -glm::normalize(simplex.vertices[0]);
+        glm::vec3 const A0 = -glm::normalize(simplex.vertices[0]);
         simplex.supportVertices[1] = gjk::Simplex::SupportVertices{
-            cso::Support(aShape,  A0),
-            cso::Support(bShape, -A0),
+            cso::Support(aShape,  A0), cso::Support(bShape, -A0),
         };
         simplex.vertices[1] = cso::Support(aShape, bShape, A0);
         ++simplex.size;
     }
 
     if (2 == simplex.size)
     {
-        glm::dvec3 const A0 = -simplex.vertices[1];
+        glm::vec3 const A0 = -simplex.vertices[1];
         uint8_t const n = (epona::fp::IsNotEqual(A0[0], 0.0) ? 0 : (epona::fp::IsNotEqual(A0[1], 0.0) ? 1 : 2));
         uint8_t const m = (n == 0 ? 1 : (n == 1 ? 2 : 1));
```

Comment:
```suggestion
        uint8_t const m = (n < 2 ? (n + 1) : 1)
```
Not sure if needed, just something that confused me at first